addons/binding/org.openhab.binding.homepilot/src/main/java/org/openhab/binding/homepilot/internal/HomePilotGatewayFactory.java

```diff
@@ -0,0 +1,11 @@
+package org.openhab.binding.homepilot.internal;
+
+import org.openhab.binding.homepilot.handler.HomePilotBridgeHandler;
+
+public class HomePilotGatewayFactory {
+
+    public static HomePilotGateway createGateway(String id, HomePilotConfig config,
+            HomePilotBridgeHandler homePilotBridgeHandler) {
+        return new DefaultHttpGateway(id, config);
+    }
+}
```

Comment:
To have the binding being picked up by the distro, you furthermore need to add it to the feature.xml, again at the alphabetically correct position. If you have a dependency on some transport bundle (e.g. upnp, mdns or serial), make sure to add a line for this dependency as well (see the other bindings as an example).

You should also add it to the parent pom.xml